src/Explorer/Explorer.tsx

```diff
@@ -1079,7 +1167,26 @@ export default class Explorer {
   }
 
   public openUploadFilePanel(parent?: NotebookContentItem): void {
-    parent = parent || this.resourceTree.myNotebooksContentRoot;
+    if (NotebookUtil.isPhoenixEnabled()) {
+      useDialog.getState().showOkCancelModalDialog(
+        Notebook.newNotebookUploadModalTitle,
+        Notebook.newNotebookModalContent,
+        "Upload",
+        async () => {
+          await this.allocateContainer();
+          parent = parent || this.resourceTree.myNotebooksContentRoot;
+          this.uploadFilePanel(parent);
+        },
+        "Cancel",
+        undefined
+      );
+    } else {
+      parent = parent || this.resourceTree.myNotebooksContentRoot;
+      this.uploadFilePanel(parent);
```

Comment:
> this.uploadFilePanel(parent); [](http://example.com/codeflow?start=5&length=30)

why upload is enabled if phoenix not enabled?

Reply:
This is for old flow. i.e., if just feature.notebooksTemporarilyDown=false

Reply:
lets chat abt this in the sync.

Given that we are not going to turn on notebooks in old infra - i dont see any point in keeping the old architecture based DE code. in fact , I am worried that it can cause problems.

Think - we should remove all that old code.
Again - dont block this checkin.
let this go and fix it immediately.